api/database/factories/PoolCandidateFactory.php

```diff
@@ -98,7 +98,22 @@ public function configure()
                 }
             }
 
-            if ($poolCandidate->education_requirement_option === EducationRequirementOption::EDUCATION->name) {
+            // set education requirement option, influenced by classification of pool
+            $classificationOne = $poolCandidate->pool->classifications()->first()->group;
+            if ($classificationOne === 'EX') {
+                $poolCandidate->update([
+                    'education_requirement_option' => EducationRequirementOption::PROFESSIONAL_DESIGNATION->name,
+                ]);
+            } else {
+                $requirementOption = $this->faker->boolean() ? EducationRequirementOption::APPLIED_WORK->name : EducationRequirementOption::EDUCATION->name;
+                $poolCandidate->update([
+                    'education_requirement_option' => $requirementOption,
+                ]);
+            }
+
+            // attach either a work or education experience to a pool candidate to meet minimum criteria
+            if ($poolCandidate->education_requirement_option === EducationRequirementOption::EDUCATION->name ||
+            $poolCandidate->education_requirement_option === EducationRequirementOption::PROFESSIONAL_DESIGNATION->name) {
```

Comment:
Three education options exist, `Professional_Designation` is for EX pool's only and so should not be seeded if that isn't the case. It creates copy bugs otherwise that'll confuse you 😛